packages/augur-tools/src/flash/scripts.ts

```diff
@@ -901,7 +970,7 @@ export function addScripts(flash: FlashSession) {
         );
       }
       this.log(`place order ${result}`);
-      // something hangs need to force exit.
+      // something is hanging. no missings await that I can see.
       process.exit;
```

Comment:
```suggestion
```

Reply:
We do this in cli.ts now so it's unnecessary here. Also it's not actually calling exit here?